Allow 0 and negative precision

sebastiansandqvist opened the pull request:
In a current project I am dealing with currencies (in cents) and need to be able to round cents to a whole dollar amount. For example, I need to get a number like `123.45` to round to `100`. In other words, negative precision.

This pull request makes negative precision possible.

sindresorhus commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "round-to",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Don't bump

sindresorhus commented on readme.md:
```diff
@@ -46,7 +46,7 @@ Type: `number`
-Type: `number` (positive integer)
+Type: `number` (integer)
 
 Number of decimal places.
```
Docs needs to be clearer about what happens when using a negative number.

sebastiansandqvist commented:
I removed the version bump and added some explanation with an example as to what happens when a negative number is used for `precision`. How does this look?

sindresorhus commented:
Looks good :)
